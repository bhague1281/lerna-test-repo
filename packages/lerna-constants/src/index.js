import * as LERNA_CONSTANTS from './lerna-constants';

export {
  LERNA_CONSTANTS
};
