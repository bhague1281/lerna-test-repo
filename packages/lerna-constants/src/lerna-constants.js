export const TEST1 = 'test1';
export const TEST2 = 'test2';
export const TEST3 = 'test3';
export const TEST4 = 'test4';
