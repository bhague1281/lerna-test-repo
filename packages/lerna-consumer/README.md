# `@bhague1281/lerna-consumer`

> TODO: description

## Usage

```
const lernaConsumer = require('@bhague1281/lerna-consumer');

// TODO: DEMONSTRATE API
```
