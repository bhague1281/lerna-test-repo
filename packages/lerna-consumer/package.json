{
  "name": "@bhague1281/lerna-consumer",
  "version": "1.0.0",
  "description": "> TODO: description",
  "author": "Brian Hague <brian.hague@sap.com>",
  "homepage": "https://github.com/bhague1281/lerna-test-repo#readme",
  "license": "ISC",
  "main": "lib/lerna-consumer.js",
  "directories": {
    "lib": "lib",
    "test": "__tests__"
  },
  "files": [
    "lib"
  ],
  "publishConfig": {
    "access": "public"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/bhague1281/lerna-test-repo.git"
  },
  "scripts": {
    "test": "jest",
    "build": "babel src --out-dir lib --copy-files"
  },
  "bugs": {
    "url": "https://github.com/bhague1281/lerna-test-repo/issues"
  },
  "devDependencies": {
    "@babel/cli": "7.2.3",
    "jest": "24.5.0"
  },
  "dependencies": {
    "@bhague1281/lerna-constants": "1.0.1"
  }
}
