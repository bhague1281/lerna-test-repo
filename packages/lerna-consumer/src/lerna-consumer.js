import {TEST1, TEST2, TEST3} from '@bhague1281/lerna-constants';

export const lernaConsumer = () => {
  return {
    meta: {
      info: 'This is a lerna consumer consuming constants'
    },
    TEST1,
    TEST2,
    TEST3
  };
};
