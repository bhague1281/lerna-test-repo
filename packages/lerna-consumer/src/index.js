import {lernaConsumer} from './lerna-consumer';

export {
  lernaConsumer
};
