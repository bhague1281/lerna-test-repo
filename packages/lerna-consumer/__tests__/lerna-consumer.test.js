import {lernaConsumer} from '../src/lerna-consumer';

describe('@bhague1281/lerna-constants', () => {
  it('should return the correct constants', () => {
    expect(lernaConsumer()).toMatchSnapshot();
  });
});
