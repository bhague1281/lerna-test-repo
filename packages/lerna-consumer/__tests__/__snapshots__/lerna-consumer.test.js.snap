// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@bhague1281/lerna-constants should return the correct constants 1`] = `
Object {
  "TEST1": "test1",
  "TEST2": "test2",
  "TEST3": "test3",
  "meta": Object {
    "info": "This is a lerna consumer consuming constants",
  },
}
`;
